api_app/analyzers_manager/observable_analyzers/tweetfeeds.py

```diff
@@ -0,0 +1,122 @@
+import json
+import logging
+import os
+from typing import Tuple
+
+import requests
+from django.conf import settings
+
+from api_app.analyzers_manager.classes import ObservableAnalyzer
+from api_app.analyzers_manager.exceptions import AnalyzerRunException
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class TweetFeeds(ObservableAnalyzer):
+    """
+    wrapper for https://tweetfeed.live
+    """
+
+    url = "https://api.tweetfeed.live/v1/"
+    filter1: str = ""
+    update_on_run: bool = True
+    time: str
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.db_dir = os.path.join(settings.MEDIA_ROOT, "TweetFeedsDBs")
+        self.create_db_dir()
+
+    def create_db_dir(self):
+        if not os.path.exists(self.db_dir):
+            os.makedirs(self.db_dir)
+
+    def location(self) -> Tuple[str, str]:
+        if self.filter1 != "":
```

Comment:
isn't enough to check whether this is populated or not instead of this comparison?